### DISCUSSION TITLE
Old commits are not recorded to my account

### DISCUSSION THREAD

**Discussion by mahmoudsebak** [community/community#22129] [New to GitHub]
<p>My user name is “mahmoudsebak” and i contributed to this project: “<a href="https://github.com/DarkGeekMS/ApeX-Server" rel="nofollow noopener">https://github.com/DarkGeekMS/ApeX-Server</a>” more than a year ago but when i see my commits now, they have my user name but they are not recorded to my account and i am not among the project’s contributors.<img src="https://user-images.githubusercontent.com/33029279/181068992-4ca780f8-7280-450e-a39c-10bf5af3dfd3.png" alt="Screenshot from 2020-06-30 17-49-51" data-base62-sha1="arK3EPrvzDQhRq3qk8WMiLYZSyO" width="689" height="67"></p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Have you reviewed the <a href="https://github.community/t/why-is-my-commit-associated-with-the-wrong-person/10199">Support ProTips article on “Why is my commit associated with the wrong person?”</a> It lists the most common reasons why problems like this are encountered.</p>
